app/presenters/notification.rb

```diff
@@ -0,0 +1,96 @@
+module ExercismWeb
+  module Presenters
+    class Notification
+      include ExercismWeb::Helpers::FuzzyTime
+
+      def self.prepare(item, current_user)
+        case item.regarding
+        when "code"
+          CodeNotification.new(item, current_user)
+        when "nitpick"
+          NitpickNotification.new(item, current_user)
+        when "like"
+          LikeNotification.new(item, current_user)
+        end
+      end
+
+      def initialize(item, current_user)
+        @notification = item
+        @current_user = current_user
+      end
+
+      def content_on_notification_page
+        if notification.item
+          html = "<li class='alert-row'>"
+          html += "<span style='padding-right: 5px;' class='fa #{icon_klass}'></span>"
+          html += large_content + " "
+          html += ago(notification.created_at)
+          html += "</li>"
+        end
```

Comment:
Let's keep all of the HTML in the `erb` file, if possible, and have each of the notification types provide all of the data that the HTML needs.